config/filter.d/postfix.conf

```diff
@@ -19,6 +19,7 @@ failregex = ^%(__prefix_line)sNOQUEUE: reject: RCPT from \S+\[<HOST>\]: 554 5\.7
             ^%(__prefix_line)sNOQUEUE: reject: VRFY from \S+\[<HOST>\]: 550 5\.1\.1 .*$
             ^%(__prefix_line)sNOQUEUE: reject: RCPT from \S+\[<HOST>\]: 450 4\.1\.8 <\S*>: Sender address rejected: Domain not found; from=<\S*> to=<\S+> proto=ESMTP helo=<\S*>$
             ^%(__prefix_line)simproper command pipelining after \S+ from [^[]*\[<HOST>\]:?$
+            ^%(__prefix_line)slost connection after \S+ from \S+\[<HOST>\]$
```

Comment:
If someone uses this filter like "out-of-the-box", it should be not affected by things, that are not really mean auth-failures / wrong handling. I mean things like port-scanning or DDOS (e. g. like `lost connection`)... 
So if we'll include it in fail2ban, then only optional (see for example the parameter `mode=aggressive` in sshd-filter).